app/src/main/java/com/iic/mokojin/activities/PlayerQueueFragment.java

```diff
@@ -68,11 +68,15 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
             }
         });
         mQueueListView.enableSwipeToDismiss();
-        scheduleUpdateClock();
 
         return rootView;
     }
 
+    @Override public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        scheduleUpdateClock();
```

Comment:
@yonbergman I moved this here after @giladgo 's recommendation, because it was called before the activity was ready when opened through the push notification. Speak now or forever hold your peace..